4.optimization/scripts/0.pairwise-compare.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[15]:
+
+
+import pathlib
+import sys
+
+import pandas as pd
+
+
+# In[16]:
+
+
+from comparators.PearsonsCorrelation import PearsonsCorrelation
+from comparison_tools.PairwiseCompareManager import PairwiseCompareManager
+
+
+# In[17]:
+
+
+# output path for bulk profiles
+output_dir = pathlib.Path("../3.preprocessing_features/data/bulk_profiles")
+output_dir.mkdir(parents=True, exist_ok=True)
+
+# extract the plate names from the file name
+plate_names = [file.stem.split("_")[0] for file in output_dir.glob("*.parquet")]
+
+for plate in plate_names:
+    output_feature_select_file = str(
+            pathlib.Path(f"{output_dir}/{plate}_bulk_feature_selected.parquet")
+        )
+    print(output_feature_select_file)
+
+
+# In[18]:
+
+
+results = []
+
+for plate in plate_names:
```

Comment:
You could also combine all of the plates into the same dataframe and specify the plate column name in `_same_columns` (e.g. `Metadata_Plate`) instead of using a for loop. This works too though, and may be a better solution if memory is a concern